mono/mini/mini-runtime.h

```diff
@@ -422,10 +423,10 @@ MONO_API void        mono_parse_env_options         (int *ref_argc, char **ref_a
 MONO_API char       *mono_parse_options_from        (const char *options, int *ref_argc, char **ref_argv []);
 MONO_API int         mono_regression_test_step      (int verbose_level, const char *image, const char *method_name);
 
-
 void                   mono_interp_stub_init         (void);
 void                   mini_install_interp_callbacks (const MonoEECallbacks *cbs);
 
+
```

Comment:
Unrelated whitespace change.

Reply:
I got confused with the white space because I was always comparing to the most recent commit. I fixed it by `git diff` 'ing with `0a51d89c4238c46f07f8b5405383547c1530809b` which was the commit before my first one. I restored the whitespace back to what it was in the aforementioned commit.

Reply:
The Github preview of the diff shows the overall change to the file, see the Files Changed tab. It looks to be like a newline is being added that wasn't there before this PR.